db/blob/blob_file_reader_test.cc

```diff
@@ -706,9 +711,10 @@ TEST_F(BlobFileReaderTest, Compression) {
     PinnableSlice value;
     uint64_t bytes_read = 0;
 
-    ASSERT_OK(reader->GetBlob(read_options, key, blob_offset, blob_size,
-                              kSnappyCompression, prefetch_buffer, &value,
-                              &bytes_read));
+    ASSERT_OK(
+        reader->GetBlob(read_options, key, blob_offset, blob_size,
+                        BuiltinCompressor::GetCompressor(kSnappyCompression),
```

Comment:
Get it once and store it?  Maybe at the top and replace the Snappy_Supported with compressor->Supported?

Reply:
Updated per suggestion (reuse compressor and use Supported method)